tests/app/inbound_sms/test_rest.py

```diff
@@ -97,8 +97,9 @@ def test_post_to_get_inbound_sms_allows_badly_formatted_number(admin_request, sa
 
 @freeze_time('Monday 10th April 2017 12:00')
 def test_post_to_get_most_recent_inbound_sms_for_service_limits_to_a_week(admin_request, sample_service):
```

Comment:
[suggestion] let's update the test name to reference "default retention" or something rather than the exact time period?